docs/datadeling/teknisk-plattform/api/rolf-api.md

```diff
@@ -0,0 +1,39 @@
+---
+image: /prosjekter/datadeling/arbeidsomrader/integrasjonsarkitektur/dokumentasjon/teknisk-plattform/figurer/rolf-picture.png
+pictureAlt: "Rolf"
+pictureCaption: "Rolf"
+title: Oppsett i Gravitee for Rolf SA
+---
+
+Rolf SA Api er en påkrevd delkomponent i UH-SAK.
+
+## Kom i gang
+
+Kortversjon:
+
+1. Log inn i Gravitee som administrator.
```

Comment:
But first: 1. Bestill oppsett av Rolf fra UiO, se [Bestill Rolf](https://www.uio.no/tjenester/it/adm-app/rolf/bestill-rolf.html). Du vil få tilsendt en JSON-fil du kan bruke videre for oppsettet.